changelog/generated/v1.29.0.md

```diff
@@ -0,0 +1,13 @@
+## v1.29.0
+
+### Changes
+
+- Update kuttl to v0.15.0 in the scorecard-test-kuttl image. ([#6401](https://github.com/operator-framework/operator-sdk/pull/6401))
+- Bump the golang base image version in the ansible-operator Dockerfiles from 1.18 to 1.19. ([#6398](https://github.com/operator-framework/operator-sdk/pull/6398))
+
+### Bug Fixes
+
+- (docs) Update the go version in the developer guide.  The documentation wasn't updated when the go version was bumped to v1.19. ([#6101](https://github.com/operator-framework/operator-sdk/pull/6101))
+- Compress the bundle content, to avoid the configMap exceed max length error. The error will look like this: 
```

Comment:
Fair enough. This would have been generated as a bug fix since that is what the changelog fragment specified it as such [here](https://github.com/operator-framework/operator-sdk/pull/6408/files#diff-9bad7aa0bf66fdf1f496160f92ae4094a091cbfec5e460abfef6a56c3bfc08e0R18)